Dockerfile

```diff
@@ -5,13 +5,16 @@ ARG NODE_ENV=production
 # Use the Node.js version specified by NODE_VERSION
 FROM node:${NODE_VERSION}-alpine AS builder
 
-RUN corepack enable && corepack prepare pnpm@latest-10 --activate
+RUN npm install -g corepack@latest
+RUN corepack enable
+RUN corepack prepare pnpm@latest-9 --activate
```

Comment:
Since we need to install corepack now - wouldn't it be easier to just run `npm i -g pnpm@9` instead?